src/theme/Footer/index.tsx

```diff
@@ -1,30 +1,57 @@
 import React from "react";
 
 import { useThemeConfig } from "@docusaurus/theme-common";
-import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
 import FooterLinks from "@theme/Footer/Links";
 import FooterLogo from "@theme/Footer/Logo";
 import FooterCopyright from "@theme/Footer/Copyright";
 import FooterLayout from "@theme/Footer/Layout";
 import { Credits, CreditsProps } from "@site/src/components/Credits";
+import Translate from "@docusaurus/Translate";
 
 function Footer(): JSX.Element | null {
   const { footer } = useThemeConfig();
-  const { siteConfig } = useDocusaurusContext();
 
   if (!footer) {
     return null;
   }
+
   const { copyright, links, logo, style } = footer;
-  const { customFields } = siteConfig;
```

Comment:
I wonder, are the custom fields removed? If not, we should dos so, so there is no confusion where the content should be defined.